server/sql_queries.py

```diff
@@ -11,7 +11,10 @@ def insert_registration_info_query(table_name):
 
     @staticmethod
     def insert_new_user_profile_query(table_name):
-        return f"INSERT INTO {table_name} (user_id) VALUES(%s)"
```

Comment:
according to what I wrote in line 123, this function is needed to save a row in table for the user after sign up